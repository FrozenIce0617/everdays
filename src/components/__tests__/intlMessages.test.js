import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import { IntlProvider } from 'react-intl';

import IntlMessage from '../intlMessages';

describe('<IntlMessage />', () => {
  it('renders component', () => {
    const component = shallow(
      <IntlProvider
        locale="en"
        messages={{
          title: 'title',
        }}
      >
        <IntlMessage id="title" />
      </IntlProvider>,
    );

    expect(shallowToJson(component)).toMatchSnapshot();
  });
});
