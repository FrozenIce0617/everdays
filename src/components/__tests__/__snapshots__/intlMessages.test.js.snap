// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<IntlMessage /> renders component 1`] = `
<InjectIntl(InjectMassage)
  id="title"
/>
`;
